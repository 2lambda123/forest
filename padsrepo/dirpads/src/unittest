#!/bin/bash

ghc -o runtest  --make -package parsec -XQuasiQuotes -XTemplateHaskell -XDeriveDataTypeable -XScopedTypeVariables      -XMultiParamTypeClasses      -XFunctionalDependencies -XFlexibleInstances      -XUndecidableInstances -XNamedFieldPuns -XRecordWildCards -XStandaloneDeriving Examples/Main.hs Examples/Second.hs
./runtest >&/tmp/test.out
if diff /tmp/test.out Data/unittest.expected
   then 
   	echo "PASSED"
   else
	echo "FAILED"
fi
rm runtest

