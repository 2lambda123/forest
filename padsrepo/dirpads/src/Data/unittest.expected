Cases: 9  Tried: 0  Errors: 0  Failures: 0Cases: 9  Tried: 1  Errors: 0  Failures: 0Cases: 9  Tried: 2  Errors: 0  Failures: 0Cases: 9  Tried: 3  Errors: 0  Failures: 0Cases: 9  Tried: 4  Errors: 0  Failures: 0Cases: 9  Tried: 5  Errors: 0  Failures: 0Cases: 9  Tried: 6  Errors: 0  Failures: 0Cases: 9  Tried: 7  Errors: 0  Failures: 0Cases: 9  Tried: 8  Errors: 0  Failures: 0                                          Cases: 9  Tried: 9  Errors: 0  Failures: 0
